ui/lib/core/addon/components/info-table-row.hbs

```diff
@@ -90,18 +87,18 @@
                 <span class="is-word-break has-text-black" data-test-row-value={{@label}}>{{@value}}</span>
               </T.Trigger>
               <T.Content @defaultClass="tool-tip">
-                <CopyButton
-                  @clipboardText={{@tooltipText}}
-                  @success={{action (set-flash-message "Data copied!")}}
-                  @tagName="div"
-                  disabled={{not @isTooltipCopyable}}
-                  class={{if @isTooltipCopyable "has-pointer"}}
-                  data-test-tooltip-copy
-                >
-                  <div class="box">
-                    {{@tooltipText}}
-                  </div>
-                </CopyButton>
+                <div class="box is-flex-center">
+                  {{@tooltipText}}
+                  {{#if @isTooltipCopyable}}
+                    <Hds::Copy::Button
+                      @text="Copy"
+                      @isIconOnly={{true}}
+                      @textToCopy={{@tooltipText}}
+                      class="hds-copy-button icon-only white-icon"
+                      data-test-tooltip-copy
+                    />
+                  {{/if}}
+                </div>
```

Comment:
Tooltip before:
<img width="245" alt="Before (copyable)" src="https://github.com/hashicorp/vault/assets/104539507/dd79f9d5-484a-4300-8225-b470d49cdc7d">
After:
<img width="335" alt="Screenshot 2023-08-28 at 10 57 40 AM" src="https://github.com/hashicorp/vault/assets/104539507/5dd0096f-1273-4948-8416-8c31c6ec48b8">
(Note that the copy button is white as per request from Design since the blue button is too hard to see. On click, the success copy button is also white instead of green)

Reply:
I'm wondering if you can use the `has-text-white` class instead of making a `white-icon` class

Reply:
I tried it out locally and `has-text-white` doesn't work since we need it specifically on the svg 😢

Reply:
Ahh that's unfortunate! Thanks for trying that out though!